composer.json

```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9",
+        "php": "^7.1.0",
```

Comment:
Well, this constraint forbids using the existing version with PHP 8. I think it is fine. I'm almost sure that the current code will *not* work with PHP 8 (PHP major versions *will* break BC for sure if we look at the existing releases where even minor versions have a hard time preserving BC fully). This way, when PHP 8 starts to come out, we can update our requirements once we start testing it with PHP 7